pages/signin/index.tsx

```diff
@@ -0,0 +1,19 @@
+import type { NextPage } from 'next';
+import { LoginPanel } from 'components/LoginPanel';
+
+const SignIn: NextPage = () => {
+  return (
+    <div
+      style={{
+        display: 'flex',
+        alignItems: 'center',
+        justifyContent: 'center',
+        height: '100vh'
+      }}
+    >
```

Comment:
Two things here:

1. I think we can switch to using the AntD components, I think its <Row justify="center" align="center"> and then wrap `<LoginPanel />` in a `<Col />`?
2. I don't think the height is needed because there is already a minHeight of `100vh` in `<NavigationBar />`